cnd/src/http_api/orderbook.rs

```diff
@@ -18,30 +19,32 @@ use uuid::Uuid;
 use warp::{http, http::StatusCode, Rejection, Reply};
 
 #[derive(Deserialize)]
-struct MakeHerc20HbitOrderBody {
+struct MakeBtcDaiBuyOrderBody {
     // TODO: Fix these names, buy/sell are incorrectly used here. Remember that changing this
     // struct is a breaking API change so the e2e tests will break.
     #[serde(with = "asset::bitcoin::sats_as_string")]
-    buy_quantity: asset::Bitcoin,
+    bitcoin_quantity: asset::Bitcoin,
     sell_token_contract: ethereum::Address,
-    sell_quantity: asset::Erc20Quantity,
+    dai_quantity: asset::Erc20Quantity,
     absolute_expiry: u32,
     refund_identity: bitcoin::Address,
     redeem_identity: identity::Ethereum,
 }
 
-impl MakeHerc20HbitOrderBody {
+impl MakeBtcDaiBuyOrderBody {
     fn to_order(&self) -> NewOrder {
         NewOrder {
-            buy: self.buy_quantity,
-            sell: asset::Erc20::new(self.sell_token_contract, self.sell_quantity.clone()),
+            pair: TradingPair::BtcDai,
+            direction: TradeDirection::Buy,
```

Comment:
Is this naming coming from Nectar? (Referring to the use of the word 'direction'.)

Reply:
Nectar uses "position" for buy/sell.